pkgs/tools/text/zoekt/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, git
+}:
+buildGoModule {
+  pname = "zoekt";
+  version = "unstable-2021-03-17";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "zoekt";
+    rev = "d92b3b80e582e735b2459413ee7d9dbbf294d629";
+    sha256 = "JdORh6bRdHsAYwsmdKY0OUavXfu3HsPQFkQjRBkcMBo=";
+  };
+
+  vendorSha256 = "d+Xvl6fleMO0frP9qr5tZgkzsnH5lPELwmEQEspD22M=";
+
+  checkInputs = [
+    git
+  ];
```

Comment:
That should be part of the buildGoModule env.

Reply:
Unfortunately, it's not:

```
=== RUN   TestFullAndShortRefNames
    tree_test.go:434: createMultibranchRepo: execution error: exit status 127, output + mkdir repo
        + cd repo
        + git init
        /bin/sh: git: not found
--- FAIL: TestFullAndShortRefNames (0.00s)
```

Reply:
Maybe its only for the buildPhase,